Nur in prefermenu-0.6.4: i18n.o.
Nur in prefermenu-0.6.4: libvdr-prefermenu.so.
Nur in prefermenu-0.6.4: libvdr-prefermenu.so.1.3.34.
diff -ru prefermenu-0.6.4-orig/preferchannel.h prefermenu-0.6.4/preferchannel.h
--- prefermenu-0.6.4-orig/preferchannel.h	2005-10-19 22:32:31.000000000 +0200
+++ prefermenu-0.6.4/preferchannel.h	2005-10-19 22:33:14.000000000 +0200
@@ -2,6 +2,7 @@
 #define __PREFERCHANNEL_H
 
 #include <vdr/plugin.h>
+#include <vdr/device.h>
 #include <vdr/channels.h>
 
 #define CHANNELSMAX 255
Nur in prefermenu-0.6.4: preferchannel.h~.
Nur in prefermenu-0.6.4: preferchannel.o.
Dateien prefermenu-0.6.4-orig/prefermenu.o und prefermenu-0.6.4/prefermenu.o sind verschieden.
diff -ru prefermenu-0.6.4-orig/preferosd.h prefermenu-0.6.4/preferosd.h
--- prefermenu-0.6.4-orig/preferosd.h	2005-10-19 22:32:31.000000000 +0200
+++ prefermenu-0.6.4/preferosd.h	2005-10-19 22:32:51.000000000 +0200
@@ -2,6 +2,7 @@
 #define __PREFEROSD_H
 
 #include <vdr/plugin.h>
+#include <vdr/device.h>
 
 static const int BORDERSIZE = 2;
 #if VDRVERSNUM >= 10307
Nur in prefermenu-0.6.4: preferosd.h~.
Nur in prefermenu-0.6.4: preferosd.o.
Nur in prefermenu-0.6.4: prefersetup.o.
